### DISCUSSION TITLE
Is it possible to rotate que model to a specific possition when clicking on a hotspot?

### DISCUSSION THREAD

**Discussion by AndresMercado** [google/model-viewer#1926] [Q&A]
Hello Google, is it possible to rotate the object to a specific possition when clicking ona hotspot, it would be cool to make the model move to display a specific area.

Thanks

**Comment by elalish** [comment]
Absolutely! Just add an `onclick` method to your hotspot that changes the `.cameraOrbit` attribute of your model-viewer element.

**Comment by AndresMercado** [comment]
Thank you, could you please share an example?

**Comment by elalish** [comment]
I should make a better example for hotspots specifically, but look at the forth one on this page to get you started: https://modelviewer.dev/examples/staging-and-camera-control.html

**Comment by AndresMercado** [comment]
Will check it out, thank you Elalish!
